internal/services/eventhub/eventhub_resource.go

```diff
@@ -499,6 +512,18 @@ func expandEventHubCaptureDescription(d *pluginsdk.ResourceData) *eventhubs.Capt
 					StorageAccountResourceId: utils.String(storageAccountId),
 				},
 			}
+
+			if destinationAuthType := destination["storage_authentication_type"]; destinationAuthType != nil && destinationAuthType.(string) != "" {
```

Comment:
We will need to update all the references to the renamed field in the resource, tests and documentation:
```suggestion
			if destinationAuthType := destination["storage_authentication"]; destinationAuthType != nil && destinationAuthType.(string) != "" {
```